packages/manager/apps/pci-block-storage/src/components/VolumeTypeTilesInput.component.tsx

```diff
@@ -0,0 +1,81 @@
+import { TilesInput, useBytes } from '@ovh-ux/manager-pci-common';
+import { useCallback, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { TVolumeModel } from '@/api/hooks/useCatalog';
+
+type Props = {
+  data?: TVolumeModel[];
+  value?: TVolumeModel;
+  onChange: (value: TVolumeModel) => void;
+  label?: string;
+  locked?: boolean;
+};
+
+export const VolumeTypeTilesInput = ({
+  data = [],
+  value,
+  onChange,
+  label = '',
```

Comment:
I make label as required to make it explicit that the label should be provided for accessibility purpose so making it optional breaks this